tests/api/test_file.py

```diff
@@ -16,6 +16,13 @@
 
 
 class FileAPITests(MontageTests):
+    @classmethod
+    def file_path(cls):
```

Comment:
Can you please make this a top-level function that takes the file name as an argument? Something like...

```
def get_file_path(filename):
    path = os.path.join(os.path.dirname(__file__), '../files', filename)
    return os.path.abspath(path)
```

Then, in the rest of the tests, we can do...

```
python = get_file_path('python-powered.png')
```

This'll make getting file paths more DRY and require less boilerplate to create when we need to.